package.json

```diff
@@ -12,28 +12,29 @@
         "start": "node packages/cli/lib/alto.js run",
         "dev": "node --inspect packages/cli/lib/alto.js run",
         "test": "pnpm -r --workspace-concurrency 1 test --verbose=true",
-        "test:spec": "./test/run-spec-tests.sh",
+        "test:spec": "./test/spec-tests/run-spec-tests.sh",
         "lint": "rome check ./",
         "lint:fix": "rome check ./ --apply",
         "format": "biome format . --write"
     },
     "devDependencies": {
-        "@biomejs/biome": "^1.5.0",
-        "@swc/core": "^1.3.56",
-        "@types/mocha": "^10.0.1",
+        "@biomejs/biome": "^1.5.1",
+        "@swc/core": "^1.3.102",
```

Comment:
not for you but I don't know why do we have this dependency, we don't use it anywhere.